yk_utils/image/image_parser.py

```diff
@@ -0,0 +1,20 @@
+"""Image parser module.
+"""
+import os
+import base64
+
+
+def parse_image(image) -> str:
```

Comment:
Missing typing.

Reply:
Here image can have multiple types (an actual image as base64 string, a file path to an image or a file-like object)